README.md

```diff
@@ -18,7 +18,12 @@ To run the selenium tests:
 1. perform initial setup `npm run protractor-setup` (once, does not
    need to be done each time)
 2. start the app in another terminal `npm start`
-3. run the tests `npm run protractor`
+3. run the tests `npm run protractor-day0` (for running day0 Installer tests)
+<<<<<<< HEAD
+4. run the tests `npm run protractor-day2` (for running day2 CLM Admin Console tests)
+=======
+4. run the tests `npm run protractor-day20` (for running day2 CLM Admin Console tests)
```

Comment:
this looks like a merge gone wrong....

Reply:
I was trying to take out extra 0 from day20 and ended up with this state. Not sure how to fix this now.

Reply:
You should be able to checkout this branch and edit this file, commit those changes and push again.

Reply:
Thanks. Fixed it.